hack/test-end-to-end.sh

```diff
@@ -374,7 +374,9 @@ validate_response "-s -k --resolve www.example.com:443:${CONTAINER_ACCESSIBLE_AP
 # Remote command execution
 echo "[INFO] Validating exec"
 registry_pod=$(oc get pod -l deploymentconfig=docker-registry -t '{{(index .items 0).metadata.name}}')
-oc exec -p ${registry_pod} whoami | grep root
+# when running as a restricted pod the registry will run with a pre-allocated
+# user in the neighborhood of 1000000+.  Look for a substring of the pre-allocated uid range
+oc exec -p ${registry_pod} id | grep 10
```

Comment:
Yeah, this grep is not quite general...let's call it a colonel grep.  Ha, @pweil- has jokes!